doc/user/bgp.rst

```diff
@@ -2693,6 +2693,11 @@ The following commands can be used in route maps:
    happen only when BGP updates have completely same communities value
    specified in the community list.
 
+.. clicmd:: match community-limit (0-65535)
+
+   This command matches BGP updates that use community list, and with a community
```

Comment:
We should also document what 0 means explicitly. Because what is now, it means unlimited, but me as an operator understand it as "do not accept the route if it has any community attached".

Reply:
whatever the meaning, I will document it. thanks for advice.